drivers/storage/portworx/manifest/remote.go

```diff
@@ -65,7 +65,7 @@ func (m *remote) downloadVersionManifest(
 	params.Add("kbver", m.k8sVersion.String())
 	u.RawQuery = params.Encode()
 
-	body, err := getManifestFromURL(u.String())
+	body, err := getManifestFromURL(u.String(), pxutil.GetPxProxyEnvVarValue(m.cluster))
```

Comment:
this function can return https first if exist, we should handle https when parsing the url as well

Reply:
this string is later used by url.Parse which works for https. if that's where you meant to handle